src/Azs.Network.Admin/Azs.Network.Admin.psd1

```diff
@@ -0,0 +1,25 @@
+@{
+  GUID = '70926bce-bc6d-44d9-9c7c-04173e27f5c9'
```

Comment:
This file should not be checked in, it gets generated and modified with the metadata specified in the readme.md

Reply:
Thanks I removed it